presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -793,6 +793,35 @@ public void testComplexPersistentCteForViewQueries()
                 "DROP View non_persistent_view");
     }
 
+    public void testCteProjectionPushDown()
```

Comment:
This test doesn't really check if the pushdowns happened, I would suggest using https://github.com/prestodb/presto/blob/3f2ed80ca1a9f464804ad6ae2cfabff3cfd5c77c/presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestLogicalCteOptimizer.java#L36

Reply:
Added a plan match test similar to the one above here:
presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestCteProjectionAndPredicatePushdown.java